rootless/specconv/specconv_linux.go

```diff
@@ -19,10 +21,13 @@ func ToRootless(spec *specs.Spec, v2Controllers []string) error {
 func getCurrentOOMScoreAdj() int {
 	b, err := ioutil.ReadFile("/proc/self/oom_score_adj")
 	if err != nil {
+		logrus.WithError(err).Warn("failed to read /proc/self/oom_score_adj")
```

Comment:
Do we know under what circumstances this fails? Would it be an `os.IsNotExist()`? (in that case, should we just ignore?)

Reply:
Never happens, IIUC